rep-0133.rst

```diff
@@ -1,6 +1,6 @@
 REP: 133
 Title: Separation of build environment and source tree tools
-Author: Tully Foote <tfoote@openrobotics.org>, Dirk Thomas, Thibault Kruse <kruset@in.tum.de>, William Woodall <wwoodall@willowgarage.com>
```

Comment:
Should probably be in a separate PR.

Reply:
The change is directly related to this PR since otherwise CI would fail. One person must have the same email address across all REPs.